cmd/manager/main.go

```diff
@@ -102,7 +102,7 @@ func main() {
 	syncPeriod := time.Hour * 3
 
 	var cacheFunc cache.NewCacheFunc
-	if len(config.WatchedNamespaces) > 1 {
+	if len(config.WatchedNamespaces) > 0 {
```

Comment:
I believe this was a different bug, it was only applying atched namespaces when more than 1 was set.

Reply:
it is actually all related